src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -2127,5 +2127,94 @@
+  "minecolonies:baker.job.desc": "Handles all wheat based recipes. Knows some special bread recipes.",
+  "minecolonies:cook.job.desc": "Handles most of the cooking and hands out food to colonists.",
+  "minecolonies:shepherd.job.desc": "Shears sheep and controls the size of the flock.",
+  "minecolonies:cowboy.job.desc": "Gets milk and meat.",
+  "minecolonies:swineherder.job.desc": "Provides the colony with pork.",
+  "minecolonies:chickenherder.job.desc": "Gathers eggs and feathers and other chicken products.",
+  "minecolonies:ranger.job.desc": "Distance fighter, high damage, low survival chance in close combat.",
+  "minecolonies:knight.job.desc": "Close quarters fighter. High resilience, low damage output.",
+  "minecolonies:composter.job.desc": "Composts a lot of the colony waste products",
+  "minecolonies:student.job.desc": "Reads books to increase their intelligence. High intelligence allows learning other skills faster.",
+  "minecolonies:archertraining.job.desc": "Archer Trainee. Practices distance fighting.",
+  "minecolonies:combattraining.job.desc": "Knight Trainee. Practices close quarter fighting.",
+  "minecolonies:sawmill.job.desc": "Provides the colony with most wood based items.",
+  "minecolonies:blacksmith.job.desc": "Provides the colony with most tools, armor and weapons.",
+  "minecolonies:stonemason.job.desc": "Provides most stone variants.",
+  "minecolonies:stonesmeltery.job.desc": "Provides the stone variants that require smelting and charcoal",
+  "minecolonies:crusher.job.desc": "Crushes cobble to gravel, gravel to sand and sand to clay.",
+  "minecolonies:sifter.job.desc": "Sifts through a variety of materials to find goodies.",
+  "minecolonies:florist.job.desc": "Plants a large variety of flowers for the colony.",
+  "minecolonies:enchanter.job.desc": "Gathers mana by observing others, enchants colonist tools and creates enchanting books.",
+  "minecolonies:researcher.job.desc": "Researches the topics the player selected.",
+  "minecolonies:healer.job.desc": "Heals wounded entities and provides cures for the sick.",
+  "minecolonies:pupil.job.desc": "Sits around and learns from the material provided by the teacher.",
+  "minecolonies:teacher.job.desc": "Sits with the kids and provides material.",
+  "minecolonies:dyer.job.desc": "Handles most of the dying that is required in the colony.",
+  "minecolonies:glassblower.job.desc": "Provides most glass based items.",
+  "minecolonies:mechanic.job.desc": "Handles redstone recipes and things other workers can't handle.",
+  "minecolonies:fletcher.job.desc": "Handles leather, string and wool based recipes.",
+  "minecolonies:planter.job.desc": "Provides sugarcane, cactus and bamboo to the colony.",
+  "minecolonies:rabbitherder.job.desc": "Handles a flock of rabbits.",
+  "minecolonies:concretemixer.job.desc": "Creates concrete powder and actual concrete",
+  "minecolonies:beekeeper.job.desc": "Administrates several beehives",
```

Comment:
I actually think this would be better as just "Keeps several beehives"  Or "Keeps bees for honey"  "administrates" sounds wrong  "Cares for several beehives" maybe?

Reply:
Put a period at the end too #inconsistent-ray